orchestration/templates/process-table.yaml

```diff
@@ -98,3 +99,102 @@ spec:
+                - name: join-table-name
+                  value: '{{ "{{tasks.diff-table.outputs.parameters.join-table-name}}" }}'
+                - name: table-name
+                  value: '{{ "{{inputs.parameters.table-name}}" }}'
+                - name: query-type
+                  value: updates
+          # deleted rows
+          - name: generate-deleted-rows-table
+            dependencies: [diff-table]
+            template: query-row-ids
+            arguments:
+              parameters:
+                - name: bq-project
+                  value: '{{ "{{inputs.parameters.staging-bq-project}}" }}'
+                - name: bq-dataset
+                  value: '{{ "{{inputs.parameters.staging-bq-dataset}}" }}'
+                - name: join-table-name
+                  value: '{{ "{{tasks.diff-table.outputs.parameters.join-table-name}}" }}'
+                - name: table-name
+                  value: '{{ "{{inputs.parameters.table-name}}" }}'
+    # Subtemplates
+    # query-full-rows will be generic for both the new rows + updates
+    - name: query-full-rows
+      inputs:
+        parameters:
+          - name: table-name
+          - name: join-table-name
+          - name: bq-project
+          - name: bq-dataset
+          - name: query-type
+      {{- include "argo.retry" . | indent 6 }}
+      script:
+        image: {{ $schemaImage }}
+        # layer of env variables to make the scripts more stand-alone and more compatible within editors
+        env:
+          - name: PROJECT
+            value: '{{ "{{inputs.parameters.bq-project}}" }}'
+          - name: DATASET
+            value: '{{ "{{inputs.parameters.bq-dataset}}" }}'
+          - name: INPUT_TABLE
+            value: '{{ "{{inputs.parameters.join-table-name}}" }}'
+          - name: TABLE
+            value: '{{ "{{inputs.parameters.table-name}}" }}'
+          - name: QUERY_TYPE
+            value: '{{ "{{inputs.parameters.query-type}}" }}'
+        command: [bash]
+        source: |
+        {{- include "argo.render-lines" (.Files.Lines "scripts/clingen-temp-table-full-rows.sh") | indent 10 }}
+
+    # query-row-ids is exclusively for deleted rows
+    - name: query-row-ids
+      inputs:
+        parameters:
+          - name: table-name
+          - name: join-table-name
+          - name: bq-project
+          - name: bq-dataset
+      {{- include "argo.retry" . | indent 6 }}
+      script:
+        image: {{ $schemaImage }}
+        env:
+          - name: PROJECT
+            value: '{{ "{{inputs.parameters.bq-project}}" }}'
+          - name: DATASET
+            value: '{{ "{{inputs.parameters.bq-dataset}}" }}'
+          - name: INPUT_TABLE
+            value: '{{ "{{inputs.parameters.input-table}}" }}'
+          - name: TABLE
+            value: '{{ "{{inputs.parameters.table-name}}" }}'
+        command: [bash]
+        source: |
+        {{- include "argo.render-lines" (.Files.Lines "scripts/clingen-temp-table-row-ids.sh") | indent 10 }}
```

Comment:
The name of the bash script should also be adjusted to be more accurate (same reasoning as for the task name)